ocpp1.6/v16.go

```diff
@@ -17,6 +19,13 @@ import (
 	"github.com/lorenzodonini/ocpp-go/ws"
 )
 
+type ChargePointConnection interface {
+	ID() string
+	TLSConnectionState() *tls.ConnectionState
+}
```

Comment:
To clarify my previous comment: if these function names match the ones on the websocket layer, then `ChargePointConnection` automatically becomes a `ws.Channel`. So on this layer we can work with a high-level interface, which just hides the underlying websocket layer, although it is the same object.

Reply:
I didn't want to break the `ws.Channel` interface, but I'll just fix it there then :grin:

Reply:
simplified it